kodilla-hibernate/src/test/java/com/kodilla/hibernate/manytomany/dao/CompanyDaoTestSuite.java

```diff
@@ -9,23 +9,73 @@
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import java.util.List;
+
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class CompanyDaoTestSuite {
     @Autowired
     CompanyDao companyDao;
+    @Autowired
+    EmployeeDao employeeDao;
+
+    private Employee johnSmith = new Employee("John", "Smith");
+    private Employee stephanieClarckson = new Employee("Stephanie", "Clarckson");
+    private Employee lindaKovalsky = new Employee("Linda", "Kovalsky");
+
+    private Company softwareMachine = new Company("Software Machine");
+    private Company dataMaesters = new Company("Data Maesters");
+    private Company greyMatter = new Company("Grey Matter");
+
+    private int softwareMachineId;
+    private int dataMaestersId;
+    private int greyMatterId;
 
     @Test
     public void testSaveManyToMany(){
+        //Given + When
```

Comment:
a cóż to? czemu to nie jest rozdzielone?

Reply:
to jest test z przykladu. W przykladzie nie bylo wydzielonej metody createTestData, ale mi sie to powtarzalo w moich testach, dlatego to wydzielilem. W przykladzie część lini było w sekcji Given a część w When.